timers.js

```diff
@@ -1,4 +1,29 @@
+  var resetButton = document.querySelector("#reset");
+  var startButton = document.querySelector("#start");
+  var pauseButton = document.querySelector("#pause");
+  var stopWatch = document.querySelector("#timer");
+  var clicks = 0;
+  var seconds = 0;
 
+  function updateTime() {
+    stopWatch.innerHTML = ("Time Elapsed: " + seconds++);
```

Comment:
This isn't wrong, but it might be better to increment the seconds on the first line, then update the html.
That would make it a bit more clear / obvious that this function is modifying the seconds variable.